framework/CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ Yii Framework 2 Change Log
 ------------------------
 
 - Bug #17744: Fix a bug with setting incorrect `defaultValue` to AR column with `CURRENT_TIMESTAMP(x)` as default expression (MySQL >= 5.6.4) (bizley)
-
+- Bug #17749: Dispatcher fix if target crashed in PHP 7.0+ (kamarton)
```

Comment:
Exact version could be determined by pull-requesting tests only and checking which versions fail.

Reply:
Would you please do it?

Reply:
sorry, I don't understand what you want (by my poor english)

Reply:
1. Create a new branch from `17749-dispatch-crash-test`.
2. Remove code from it except unit tests.
3. Make another pull request.
4. Wait till travis is finished. It will show you which PHP versions fail and which did not.

Reply:
#17768

Reply:
OK. That seems to be 7.0+. Thanks for validating it.